### PR TITLE
docs(svelte quickstart) svelte demo

### PR DESCRIPTION
**Pull request by abeldebruijn**
## What kind of change does this PR introduce?

I created a svelte quickstart demo which is similar to [this](https://supabase.io/docs/guides/with-react)

## What is the new behavior?

Screenshot 2021-06-07 at 11 46 20

## Additional context

I implemented this feature in about an hour after watching the build-in public youtube series from supabase. [watch here](https://www.youtube.com/playlist?list=PL5S4mPUpp4OspvfNwUA35xW1V8yUyj2fz). In my opinion, it wasn't really necessary to create an issue, because of the quick development. This is my first supabase contribution, so I hope I have managed to comply with all guidelines. Any suggestions are welcome

### CODE REVIEW

**Comment by dawnofmidnight** [comment]
In the navbar, I think that "Quickstart: Svelte" should be above "Self Hosting" so that it's grouped with the other Quickstarts
image

File: web/docs/guides/with-svelte.mdx [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+    <div>
+      <input type="submit" class='button block' value={loading ? "Loading" : "Send magic link"} disabled={loading} />
+    </div>
+  </div>
+</form>
+```
+
+### User store
+To access the user information in other places, we use writable store. Create a new file called `sessionStore.js`
+```javascript title="src/sessionStore.svelte"
```
Comment (kiwicopple) [inline web/docs/guides/with-svelte.mdx:272]: Should this one be `src/sessionStore.js`?

File: web/docs/guides/with-svelte.mdx [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+</form>
+```
+
+
+
+### Launch!
+
+Now that we have all the components in place, let's update `App.svelte`:
+
+```html title="src/App.js"
```
Comment (kiwicopple) [inline web/docs/guides/with-svelte.mdx:398]: should be src/App.svelte?

File: web/docs/guides/with-svelte.mdx [hunk 3]
```diff
@@ -0,0 +1,550 @@
...
+<script>
+	import {user} from "./sessionStore"
+	import {supabase} from "./supabaseClient"
+	import Auth from "./Auth.svelte"
+	import Profile from "./Profile.svelte"
```
Comment (kiwicopple) [inline web/docs/guides/with-svelte.mdx:403]: This one needs to be Account.svelte

**Review by kiwicopple — APPROVED** [review]
Looks awesome. Thanks for dropping in on the livestream

File: web/sidebars.js [hunk 4]
```diff
@@ -82,7 +82,7 @@ module.exports = {
       type: 'category',
       label: 'Guides',
       collapsed: false,
-      items: ['guides/examples', 'guides/with-nextjs', 'guides/with-react', 'guides/self-hosting'],
+      items: ['guides/examples', 'guides/with-nextjs', 'guides/with-react', 'guides/self-hosting', 'guides/with-svelte'],
```
Comment (kiwicopple) [inline web/sidebars.js:85]: Could you please put `with-svelte` before `self-hosting`? thanks!

**Pull request merged**